app/models/account.rb

```diff
@@ -46,6 +47,18 @@ class Account < ApplicationRecord
   has_many :groups
   has_many :group_owners
 
+  # FIXME(ezekg) roles should have an account_id foreign key
+  has_many :environment_roles, through: :environments, source: :role
```

Comment:
Also need to export permissions and event types.

Reply:
Maybe we can add an `.exports with: -> { ... }` method so that some models can add additional attributes to export, e.g. roles, permissions, etc.